versions.sh

```diff
@@ -179,6 +179,9 @@ for version in "${versions[@]}"; do
 			version: env.fullVersion,
 			pip: {
 				version: env.pipVersion,
+			},
+			"get-pip": {
+				version: env.getPipCommit,
```

Comment:
Maybe this should be a full URL like https://github.com/pypa/get-pip/commit/12b30d8c229961b6c0892101b365ec185fab7e14 so GitHub shortens them to be prettier?

Reply:
Yeah, full URL seems nicer.

Reply:
Updated!